code/modules/surgery/amputation.dm

```diff
@@ -14,7 +14,7 @@
 
 
 /datum/surgery_step/sever_limb
-	name = "sever limb"
+	name = "sever limb (shears/scalpel/saw)"
```

Comment:
I don't think we should mention shears here since you have to go out of your way to get them, especially not first

Reply:
i think it should only show the correct tool and not scapel/saw